packages/account/src/Sections/Security/Passwords/__tests__/deriv-password.spec.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import DerivPassword from '../deriv-password';
+import { WS } from '@deriv/shared';
+
+jest.mock('Assets/ic-brand-deriv-red.svg', () => () => <></>);
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    WS: {
+        verifyEmail: jest.fn(),
+    },
+}));
+
+describe('<DerivPassword />', () => {
+    let mock_props = {
+        email: 'mf@deriv.com',
+        is_social_signup: false,
+        social_identity_provider: undefined,
+        landing_company_shortcode: 'maltainvest',
+    };
+
+    let modal_root_el;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+    test('Should render properly', async () => {
+        render(<DerivPassword {...mock_props} />);
+        expect(
+            screen.getByRole('heading', {
+                name: /deriv password/i,
+            })
+        ).toBeInTheDocument();
+    });
+
+    test('displays the correct platform information for non-MF clients', () => {
+        const { container } = render(<DerivPassword {...mock_props} landing_company_shortcode='svg' />);
```

Comment:
Can this be tested without testing the presence of classes?

Reply:
updated